db-commons/src/main/scala/org/bitcoins/db/DbCommonsColumnMappers.scala

```diff
@@ -166,6 +188,108 @@ abstract class DbCommonsColumnMappers {
+  }
+
+  implicit val dlcTimeoutsMapper: BaseColumnType[DLCTimeouts] = {
+    MappedColumnType.base[DLCTimeouts, String](
+      timeouts =>
+        timeouts.penaltyTimeout.toString ++ "|" ++ timeouts.contractMaturity.mkString ++ "|" ++ timeouts.contractTimeout.mkString,
+      str => {
+        val strings = str.split('|')
+        DLCTimeouts(
+          strings(0).toInt,
+          BlockStamp
+            .fromString(strings(1))
+            .get
+            .asInstanceOf[BlockStampWithFuture],
+          BlockStamp
+            .fromString(strings(2))
+            .get
+            .asInstanceOf[BlockStampWithFuture]
+        )
+      }
+    )
+  }
+
+  implicit val cetSigsMapper: BaseColumnType[CETSignatures] = {
+    MappedColumnType.base[CETSignatures, String](
+      cetSigs =>
+        "" ++ cetSigs.winSig.hex ++ "|" ++ cetSigs.loseSig.hex ++ "|" ++ cetSigs.refundSig.hex,
+      str => {
+        val strings = str.split('|')
+
+        val winSig =
+          InputPSBTRecord.fromHex(strings(0)).asInstanceOf[PartialSignature]
+        val loseSig =
+          InputPSBTRecord.fromHex(strings(1)).asInstanceOf[PartialSignature]
+        val refundSig =
+          InputPSBTRecord.fromHex(strings(2)).asInstanceOf[PartialSignature]
+
+        CETSignatures(winSig, loseSig, refundSig)
+      }
+    )
+  }
+
+  implicit val fundingSigsMapper: BaseColumnType[FundingSignatures] = {
+    MappedColumnType.base[FundingSignatures, String](
+      fundingSigs => fundingSigs.sigs.foldLeft("")(_ ++ _.hex ++ "|"),
+      str => {
+        val strings = str.split('|')
+        val sigs = strings
+          .map(hex =>
+            InputPSBTRecord.fromHex(hex).asInstanceOf[PartialSignature])
+          .toVector
+
+        FundingSignatures(sigs)
+      }
+    )
+  }
+
+  implicit val feeUnitMapper: BaseColumnType[FeeUnit] = {
+    MappedColumnType
+      .base[FeeUnit, String](
+        _.currencyUnit.hex,
+        hex => SatoshisPerVirtualByte(Satoshis.fromHex(hex)))
+  }
+
+  implicit val fundingInputsMapper: BaseColumnType[
+    Vector[TransactionOutPoint]] = {
+    MappedColumnType.base[Vector[TransactionOutPoint], String](
+      _.foldLeft("")(_ ++ _.hex),
+      _.grouped(72).map(TransactionOutPoint.fromHex).toVector)
+  }
+
+  implicit val remoteFundingInputsMapper: BaseColumnType[
+    Vector[(TransactionOutPoint, TransactionOutput)]] = {
+    MappedColumnType
+      .base[Vector[(TransactionOutPoint, TransactionOutput)], String](
+        _.foldLeft("") {
+          case (str, (outpoint, output)) =>
+            str ++ outpoint.hex ++ output.hex
+        },
+        _.grouped(144)
+          .map(segment =>
+            (TransactionOutPoint(segment.take(72)),
+             TransactionOutput(segment.drop(72))))
+          .toVector
+      )
+  }
+
+  implicit val networkMapper: BaseColumnType[BitcoinNetwork] = {
+    MappedColumnType.base[BitcoinNetwork, String](
+      _.name,
+      Networks.fromString(_).get.asInstanceOf[BitcoinNetwork]
```

Comment:
Should probably add a `BitcoinNetwork.fromString` and have `Networks.fromString` delegate